src/algorithm/evaluation.py

```diff
@@ -391,6 +402,50 @@ def parse_answer_column(answer_column):
             parsed_answer = answer_column
         return parsed_answer
 
+    @staticmethod
+    def get_colors(verdict_colors, verdict_symbol_colors, draw_answer_groups):
+        verdict_colors_dict = {
+            "correct": MathUtils.hex_to_bgr(verdict_colors["correct"]),
+            "neutral": MathUtils.hex_to_bgr(verdict_colors["neutral"]),
+            "negative": MathUtils.hex_to_bgr(verdict_colors["negative"]),
+            "bonus": MathUtils.hex_to_bgr(verdict_colors["bonus"]),
+        }
+        verdict_symbol_colors_dict = {
+            "positive": MathUtils.hex_to_bgr(verdict_symbol_colors["positive"]),
+            "neutral": MathUtils.hex_to_bgr(verdict_symbol_colors["neutral"]),
+            "negative": MathUtils.hex_to_bgr(verdict_symbol_colors["negative"]),
+            "bonus": MathUtils.hex_to_bgr(verdict_symbol_colors["bonus"]),
+        }
+
+        draw_answer_groups_dict = {
+            "enabled": draw_answer_groups["enabled"],
+        }
+        if draw_answer_groups["enabled"]:
+            draw_answer_groups_dict["color_sequence"] = [
+                MathUtils.hex_to_bgr(hex)
+                for hex in draw_answer_groups["color_sequence"]
+            ]
+
+        return verdict_colors_dict, verdict_symbol_colors_dict, draw_answer_groups_dict
+
+    def get_draw_question_dict(self):
+        (
+            verdict_colors,
+            verdict_symbol_colors,
+            draw_answer_groups,
+        ) = map(
+            self.draw_question_verdicts.get,
+            [
+                "verdict_colors",
+                "verdict_symbol_colors",
+                "draw_answer_groups",
+            ],
+        )
+
+        self.verdict_colors, self.verdict_symbol_colors, self.draw_answer_groups = (
+            self.get_colors(verdict_colors, verdict_symbol_colors, draw_answer_groups)
+        )
```

Comment:
Let's keep the body of get_colors inside this function itself since it's used only once